### ISSUE TITLE
Changing 'value_track_width' of a slider made "value_track bar" not to be center

### ISSUE THREAD

**Issue by BorisFR** [kivy/kivy#6671]
## Versions

* Python: 3.7.1
* OS: macosx (darwin) 10.15.2
* Kivy: 1.11.1
* Kivy installation method: python3 -m pip install kivy

### Description

In a slider, the value_track bar is not centered when changing value_track_width value.

### Code and Logs

```python
        Slider:
            id: theVolume
            size_hint: (None, None)
            height: self.parent.height
            width: self.parent.width / 3.33
            background_width: self.height / 5.0
            cursor_image: 'slider.png'
            cursor_width: self.height / 1.5
            cursor_height: self.height / 1.5
            #sensitivity: 'handle'
            min: 0
            max: 100
            step: 1
            orientation: 'horizontal'
            value: 50
            value_track: True
            value_track_width: self.height / 10.0
            value_track_color: [1, 0.8, 0, 1]

```
![Capture d’écran 2019-12-29 à 14 24 49](https://user-images.githubusercontent.com/1536092/71557473-08fc0480-2a47-11ea-802f-396e3447672f.png)

**Comment by DanTheMan2000** [comment]
I may have found and fix this problem and pull request #6579

**Comment by matham** [comment]
Merged the PR. Sorry for the delay.
